### ISSUE TITLE
Version 0.7.4 always crashes after break

### ISSUE THREAD

**Issue by raqua** [tom-james-watson/breaktimer-app#58]
17:46:02.299 › Checking for updates...
17:46:02.311 › APPIMAGE env is not defined, current application is not an AppImage
17:46:17.443 › Send event PLAY_START_GONG []
17:46:17.922 › GET_SETTINGS
17:46:17.924 › GET_SETTINGS
17:46:17.926 › GET_BREAK_END_TIME
17:46:17.930 › GET_BREAK_END_TIME
17:48:17.984 › Send event PLAY_START_GONG []
Segmentation fault

I am on Linux Mint 18.3. Version 0.6.2 worked fine before.

**Comment by tom-james-watson** [comment]
What package format of the app are you running? deb?

**Comment by raqua** [comment]
Yes. deb.

**Comment by tom-james-watson** [comment]
I can't recreate this. Does the problem persist after a system reboot?

If so, do you mind trying the deb from 0.7.3? https://github.com/tom-james-watson/breaktimer-app/releases/tag/v0.7.3

**Comment by raqua** [comment]
Yes, version 0.7.3 works fine, 0.7.4 not, even after reinstall.

**Comment by chilons** [comment]
Same problem on Ubuntu 20.04

**Comment by cmfcmf** [comment]
For me it crashes only after some breaks, not all of them (Ubuntu 20.04, deb, version 0.7.4).

**Comment by tom-james-watson** [comment]
Not really sure how to approach debugging this to be honest.

Maybe just bumping core dependencies to latest versions might fix it.

**Comment by iamjinme** [comment]
Same bug in Elementary OS 5.1.6 Hera: `Linux rappi 4.15.0-109-generic #110-Ubuntu SMP Tue Jun 23 02:39:32 UTC 2020 x86_64 x86_64 x86_64 GNU/Linux`

**Comment by tom-james-watson** [comment]
I've just pushed a version 0.7.5 that upgrades core dependencies such as electron. Hopefully that fixes the issue.

Please upgrade to this version and let me know if you still see any crashing. Thanks!

**Comment by cmfcmf** [comment]
I've seen no more crashes since using 0.7.5 :tada:

**Comment by tom-james-watson** [comment]
Great! I'm going to tentatively close this. If crashes are still seen we can reopen or open a new issue.
